portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -52,7 +64,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the input from the comments form
     String authorText = getParameter(request, "author");
     String valueText = getParameter(request, "value");
-    comments.add(new Comment(authorText, valueText));
+    Comment comment = new Comment(authorText, valueText);
+
+    // store the comments in a datastore
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("author", comment.getAuthor());
+    commentEntity.setProperty("value", comment.getValue());
+    commentEntity.setProperty("timestamp", comment.getTimestamp());
```

Comment:
This could be placed into a method that takes the request and returns an Entity

Reply:
Good idea, added as a method to the new CommentDatabase class.